scripts/send_log_config.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
```

Comment:
This file is under `scripts`? Isn't this a utility a server administrator for a CodeChecker server could use to manage configuration runtime? If this is only under `scripts` and never installed anywhere, then this is just a "source" code, not a "deployed" code, and thus, server owners have to have the source package checked out (not usual behaviour for, e.g. binary packages) to access this functionality.

Reply:
I did not plan to ship this script with the built package. It is an administrator tool which is not needed by every user (If I'm wrong we can put it into the package).